tests/publiccloud/prepare_tools.pm

```diff
@@ -102,11 +107,16 @@ sub run {
     # Create some directories, ipa will need them
     assert_script_run("img-proof list");
     record_info('img-proof', script_output('img-proof --version'));
+    my $terraform_version = '0.14.1';
```

Comment:
something like
```
my $terraform_version = get_var('TERRAFORM_VERSION',  '0.14.1');
```

Reply:
let's keep this discussion in one place . already answered see above